packages/lesswrong/components/async/EditorForm.jsx

```diff
@@ -19,14 +19,25 @@ import { myKeyBindingFn } from './editor-plugins/keyBindings.js'
 import createLinkifyPlugin from './editor-plugins/linkifyPlugin'
 import ImageButton from './editor-plugins/image/ImageButton.jsx';
 import { Map } from 'immutable';
+import { withStyles } from '@material-ui/core/styles';
+import NoSsr from '@material-ui/core/NoSsr';
+import compose from 'lodash/flowRight';
+
 import {
   createBlockStyleButton,
   ItalicButton,
   BoldButton,
   UnderlineButton,
   BlockquoteButton,
 } from 'draft-js-buttons';
-import NoSsr from '@material-ui/core/NoSsr';
+
+const styles = theme => ({
+  placeholder: {
+    color: theme.palette.grey[500],
+    position: "absolute",
```

Comment:
This positioning goes awry on the new-sequence page, because the placeholder's parent isn't positioned.